### ISSUE TITLE
Notifier URL validation bug with shoutrrr

### ISSUE THREAD

**Issue by dmontoya3** [hay-kot/homebox#396]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### Homebox Version

Version: v0.9.0

### What is the issue you are experiencing?

Currently having two separate issues with the shoutrrr notifications configuration

Trying with IFTTT webhooks

In homebox I place the name and URL
![image](https://user-images.githubusercontent.com/41489098/230152176-aab6cdad-a884-4742-8c50-6e606befc4c3.png)

When I test it. HomeBox says "Failed to test notifier" but it actually works. IFTTT Gets the web request and process it accordingly. Then when submiting it, HomeBox returns an error saying "Failed to create notifier"

Home Assistant
I'm trying also with Home Assistant. In this case the test says it was succesful and it did work.
![image](https://user-images.githubusercontent.com/41489098/230152220-f8c9d2a4-4154-40b3-a901-e6ded8740595.png)

But when I submit it, again HomeBox says "Failed to create notifier"
![image](https://user-images.githubusercontent.com/41489098/230150012-cb57a40f-2e1a-46db-839c-ba18f5edf0ca.png)

Logs show this
```
4:25PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=GET path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003217
4:25PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=GET path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003217 status=200
4:28PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=POST path=/api/v1/notifiers/test rid=52a8d8a86b08/A0dTfjmiUQ-003218
4:28PM ERR ../go/src/app/internal/web/mid/errors.go:31 > ERROR occurred error="failed to send IFTTT event \"homebox_notification\": got response status code 200 OK" req_id=52a8d8a86b08/A0dTfjmiUQ-003218
4:28PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=POST path=/api/v1/notifiers/test rid=52a8d8a86b08/A0dTfjmiUQ-003218 status=500
4:32PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=POST path=/api/v1/notifiers/test rid=52a8d8a86b08/A0dTfjmiUQ-003219
4:32PM ERR ../go/src/app/internal/web/mid/errors.go:31 > ERROR occurred error="failed to send IFTTT event \"homebox_notification\": got response status code 200 OK" req_id=52a8d8a86b08/A0dTfjmiUQ-003219
4:32PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=POST path=/api/v1/notifiers/test rid=52a8d8a86b08/A0dTfjmiUQ-003219 status=500
4:34PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=POST path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003220
4:34PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=POST path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003220 status=201
4:34PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=GET path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003221
4:34PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=GET path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003221 status=200
4:35PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=POST path=/api/v1/notifiers/test rid=52a8d8a86b08/A0dTfjmiUQ-003222
4:35PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=POST path=/api/v1/notifiers/test rid=52a8d8a86b08/A0dTfjmiUQ-003222 status=200
4:35PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=POST path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003223
4:35PM ERR ../go/src/app/internal/web/mid/errors.go:31 > ERROR occurred error="[{\"field\":\"URL\",\"error\":\"Key: 'NotifierCreate.URL' Error:Field validation for 'URL' failed on the 'shoutrrr' tag\"}]" req_id=52a8d8a86b08/A0dTfjmiUQ-003223
4:35PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=POST path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003223 status=422
4:35PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=GET path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003224
4:35PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=GET path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003224 status=200
4:37PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=POST path=/api/v1/notifiers/test rid=52a8d8a86b08/A0dTfjmiUQ-003225
4:37PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=POST path=/api/v1/notifiers/test rid=52a8d8a86b08/A0dTfjmiUQ-003225 status=200
4:37PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=POST path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003226
4:37PM ERR ../go/src/app/internal/web/mid/errors.go:31 > ERROR occurred error="[{\"field\":\"URL\",\"error\":\"Key: 'NotifierCreate.URL' Error:Field validation for 'URL' failed on the 'shoutrrr' tag\"}]" req_id=52a8d8a86b08/A0dTfjmiUQ-003226
4:37PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=POST path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003226 status=422
4:37PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=GET path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003227
4:37PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=GET path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003227 status=200
4:43PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=POST path=/api/v1/notifiers/test rid=52a8d8a86b08/A0dTfjmiUQ-003228
4:43PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=POST path=/api/v1/notifiers/test rid=52a8d8a86b08/A0dTfjmiUQ-003228 status=200
4:44PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=POST path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003229
4:44PM ERR ../go/src/app/internal/web/mid/errors.go:31 > ERROR occurred error="[{\"field\":\"URL\",\"error\":\"Key: 'NotifierCreate.URL' Error:Field validation for 'URL' failed on the 'shoutrrr' tag\"}]" req_id=52a8d8a86b08/A0dTfjmiUQ-003229
4:44PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=POST path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003229 status=422
4:44PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=GET path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003230
4:44PM INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=GET path=/api/v1/notifiers rid=52a8d8a86b08/A0dTfjmiUQ-003230 status=200
4:44PM INF ../go/src/app/internal/web/mid/logger.go:25 > request received method=GET path=/api/v1/locations/tree rid=52a8d8a86b08/A0dTfjmiUQ-003231
```

### How can the maintainer reproduce the issue?

For IFTTT
Go to profile, on the notifiers section, click create. In name placed "IFTTT Notifications" and in URL place
ifttt://PLACEYOURWEBHOOKID/?events=homebox_notification&value1=value1&value2=value2&value3=value3

IFTTT should be preconfigured to receive such webhook and event name

For HomeAssistant
Go to profile, on the notifiers section, click create. In name placed "Home Assisntant Notifications" and in URL place
generic://PLACEYOURHOMASSISTANTIPADDRESS/api/webhook/homebox_notification_hassio?template=json&disabletls=yes

Home Assistant should be preconfigured to receive such webook and event name

In both scenarios, the webhook trigger is received and processed correctly. However it can't be saved within HomeBox after testing

### Deployment

Docker (Linux)

**Comment by hay-kot** [comment]
> ifttt://PLACEYOURWEBHOOKID/?events=homebox_notification&value1=value1&value2=value2&value3=value3

Works fine for me,

> generic://PLACEYOURHOMASSISTANTIPADDRESS/api/webhook/homebox_notification_hassio?template=json&disabletls=yes

Doesn't work - adding a fix shortly.

**Comment by hay-kot** [comment]
Assuming this was fixed in #422
